index.html

```diff
@@ -80,6 +80,10 @@ <h2 class="name">Django</h2>
       </div>
     </div>
   </main>
-  <footer class="train-footer"></footer>
+  <footer class="train-footer">
+    <p class="txt">WE ARE GAME CHANGERS</p>
+    <img class="logo" src="assets/wolox_logo_white.svg" alt="WOLOX">
+    <small class="copyright">Copyright 2018. Wolox. All rights reserved.</small>
```

Comment:
as long as you don't use the css attached to the `small` tag i really linked. I looked up it's usage
```
HTML5 the element is intended to be used to identify items of secondary importance such as copyright notices, side comments, and legal notices.
```
so it looks like a good idea to use it here. The only thing that I don't like about it is that it adds style to the content and that should be done through css and not from the html